src/main/java/io/aiven/kafka/connect/common/output/jsonwriter/ValueBuilder.java

```diff
@@ -43,8 +43,8 @@ public ValueBuilder() {
      * Takes the {@link SinkRecord}'s value as a JSON.
      *
      * @param record        the record to get the value from
-     * @throws DataException when the value is not actually a JSON
      * @return JsonNode     Value transformed to any JSON value
+     * @throws DataException when the value is not actually a JSON
```

Comment:
New checkstyle requires this change because of this rule 
https://checkstyle.sourceforge.io/config_javadoc.html#AtclauseOrder